src/components/modal-portal/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+// probably use a prop instead of static/pre-defined mount root?
+const modalRoot = document.getElementById('modal-root');
+
+/**
+ * Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+ * @see https://reactjs.org/docs/portals.html
+ */
+export default class ModalPortal extends React.Component {
```

Comment:
this is a bit weird because although we define a portal for the modal, it is not rendering inside the portal because the reactstrap modal already uses a portal (https://github.com/reactstrap/reactstrap/blob/master/src/Modal.js). so its like we are opening a portal which opens another portal. not necessarily a problem, just feels weird and possibly unnecessary

Reply:
deleted